src/main/plugins/org.dita.pdf2/xsl/fo/links.xsl

```diff
@@ -407,9 +406,7 @@ See the accompanying license.txt file for applicable licenses.
           <fo:list-item xsl:use-attribute-sets="related-links.ul.li">
             <fo:list-item-label xsl:use-attribute-sets="related-links.ul.li__label">
               <fo:block xsl:use-attribute-sets="related-links.ul.li__label__content">
-                <fo:inline>
```

Comment:
For all of the syntax groups below, there is no containing element (though I'm guessing those are rarely used). Today, this construct is probably (most often) just resulting in the extra markup for link elements.